cleanlab/experimental/README.md

```diff
@@ -2,7 +2,7 @@
 
 Methods in this `experimental` module are bleeding edge and may have sharp edges. They are not guaranteed to be stable between different cleanlab versions.
 
-Some of these files include various models that can be used with cleanlab to find issues in specific types of data. These require dependencies on deep learning and other machine learning packages that are not official cleanlab dependencies. You must install these dependencies on your own if you wish to use them.
+Some of these files include various models that can be used with cleanlab to find issues in specific types of data. These require dependencies on deep learning and other machine learning packages that are not official cleanlab dependencies. You must install these dependencies on your own if you wish to use them.
```

Comment:
I would still list label_issues_batched in the list below with a couple word summary about what it does (just with no dependencies).
Maybe change title of list to "The files and their dependencies are as follows:"